diffengine/twitter.py

```diff
@@ -16,19 +16,21 @@ class TwitterHandler:
     consumer_key = None
     consumer_secret = None
 
-    def __init__(self, consumer_key, consumer_secret):
-        if not consumer_key or not consumer_secret:
+    def __init__(self, config):
+        if not all(["consumer_key" in config, "consumer_secret" in config,
+                    "token" in config]):
```

Comment:
Remember the token is peer feed!

Reply:
Now it is clear! I was a bit confused on the need of the token per feed, as I thought it was only dependant on the consumer secret and consumer key. I had only be consuming and not tweeting via the API in the past. 

I'll get the token bakck to the tweet diff.